src/Xpress.jl

```diff
@@ -28,20 +28,64 @@ const libxprs = joinpath(
 include("Lib/Lib.jl")
 include("helper.jl")
 include("api.jl")
-include("license.jl")
+include("MOI/MOI_wrapper.jl")
+include("MOI/MOI_callbacks.jl")
+
+function _get_xpauthpath(; verbose::Bool)
+    candidates = String[pwd(), dirname(libxprs)]
+    for key in ("XPAUTH_PATH", "XPRESS_DIR")
+        if haskey(ENV, key)
+            push!(candidates, replace(ENV[key], "\"" => ""))
+        end
+    end
+    push!(candidates, joinpath(dirname(dirname(libxprs)), "bin"))
+    for candidate in candidates
+        for filename in (candidate, joinpath(candidate, "xpauth.xpr"))
+            if isfile(filename)
+                if verbose
+                    @info("Xpress: Found license file $filename")
+                end
+                return filename
+            end
+        end
+    end
+    return error(
+        "Could not find xpauth.xpr license file. Set the `XPRESSDIR` or " *
+        "`XPAUTH_PATH` environment variables.",
+    )
+end
 
 function initialize()
     Libdl.dlopen(libxprs)
-    userlic()
```

Comment:
all our code relies on calling this. I would not like to deprecate unless extremely necessary.